release_tester/selenium_ui_test/pages/service_page.py

```diff
@@ -172,21 +173,12 @@ def select_category_option_search_filter(self, keyword):
 
     def select_demo_geo_s2_service(self, max_retries=3):
         """Selecting demo geo s2 service from the list"""
-        for attempt in range(max_retries):
-            try:
-                self.webdriver.refresh()
-                self.wait_for_ajax()
-                
-                self.tprint('Selecting demo_geo_s2 service \n')
-                geo_service = "//*[text()='demo-geo-s2']"
-                geo_service_sitem = self.locator_finder_by_xpath(geo_service, 10, 2)
-                geo_service_sitem.click()
-                time.sleep(2)
-            except (TimeoutException, NoSuchElementException, ElementNotInteractableException) as e:
-                self.progress(f"Attempt {attempt + 1}: Exception occurred - {e}, retrying...")
-            
-            # Wait a bit before retrying
-            time.sleep(2)
+        self.webdriver.refresh()
```

Comment:
this looks like an errnous revert to me? Please confirm.

Reply:
You are right Willi. It happened mistakenly during resolving conflict. I updated it again with some clear error logs.